src/main/java/qupath/ext/instanseg/core/InstanSeg.java

```diff
@@ -653,6 +658,27 @@ public Builder outputAnnotations() {
             return this;
         }
 
+        /**
+         * Set a number of optional arguments
+         * @param optionalArgs The argument names and values.
+         * @return A modified builder.
+         */
+        public Builder args(Map<String, Object> optionalArgs) {
```

Comment:
Can this take `Map<String, ?>` instead? It should be possible to use a `Map<String, Number>` (for example) without the compiler complaining.

Reply:
I started with that, but got some horrific compiler error about cap1 being ? Extends object and cap2 being ? Extends objects, these two apparently being incompatible

Reply:
I think copying to a separate map *is* better, and it's fine if it's `Map<String, Object>` - it's only the method parameter that needs to be generously accepting with regard to whatever the second generic parameter is. Did a quick check and it should compile fine if you *only* change the method variable type and keen the rest as it was.